### ISSUE TITLE
/platform/GroupV2/Search not returning list of clans that match search parameter

### ISSUE THREAD

**Issue by kolkonos** [Bungie-net/api#728]
This end point is described as capable of returning a list of groups given a partial name string, however, when searching on a groupType = 1 only exact matches are returned, which is basically what the GetGroupByName does.

I believe this is a bug.

**Comment by vthornheart-bng** [comment]
Yes, this is unfortunately a long-standing bug due to underlying Clans systems in the game to which we connect.  Let me check on the status of it - I'm unsure if it is something that is ever going to be fixed, or if we're basically going to be forced to have exact match exclusively for the forseeable future.  You can actually see it on our site too: we had to turn our "clan search" functionality into an exact match search due to these same underlying issues.

I'll see what I can find out, but given that it's been a year now I get the feeling that it's fallen into the "likely never to get fixed" portion of the backlog.

**Comment by vthornheart-bng** [comment]
Ah, that was a quick response.  Indeed, it's unfortunately not something that's on the docket to be fixed given other priorities on the backlog for the teams working in that area.  Sorry about that!

**Comment by floatingatoll** [comment]
Does any API documentation need to be updated to reflect this?
